lisa/src/main/java/it/unive/lisa/cfg/type/Type.java

```diff
@@ -0,0 +1,72 @@
+package it.unive.lisa.cfg.type;
+
+/**
+ * Type interface.
```

Comment:
We should make it explicit (probably here) that instances of this interface should be unique to help implementers.